datahub-web-react/README.md

```diff
@@ -1,3 +1,124 @@
-This module contains an incubating React App. Please note that this is *not* production-ready and is still in beta. Our initial milestone is to achieve functional parity with the Ember web client. 
+DataHub React Client (Incubating)
```

Comment:
Would prefer to keep the terminology "React Application" across the doc ... even though technically there is a Client here...